### ISSUE TITLE
PDF Indexer

### ISSUE THREAD

**Issue by pdvsofismo** [BITPlan/com.bitplan.simplegraph#12]
This module is used to index a series of pdf files with a pre-defined set of keywords

**Comment by pdvsofismo** [comment]
http://wiki.bitplan.com/index.php/SimpleGraph-PDF

**Comment by WolfgangFahl** [comment]
Yes a combination of the FileSystem, PDF and Excel Module should be able to accomplish what you'd like to do.
